src/main/java/net/nextencia/rrdiagram/Server.java

```diff
@@ -0,0 +1,264 @@
+package net.nextencia.rrdiagram;
+
+import net.nextencia.rrdiagram.common.Utils;
+import net.nextencia.rrdiagram.grammar.model.*;
+import net.nextencia.rrdiagram.grammar.rrdiagram.RRDiagram;
+import net.nextencia.rrdiagram.grammar.rrdiagram.RRDiagramToSVG;
+
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import java.net.InetSocketAddress;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.Executors;
+import java.util.logging.Logger;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+import java.util.logging.Handler;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.Map;
+import java.util.HashMap;
+import java.net.URLDecoder;
+import java.net.URI;
+import java.util.logging.Level;
+import java.util.logging.LogManager;
+
+class Helper {
+    public static boolean writeHTTPResponse(HttpExchange httpExchange, int responseCode, String content, String contentType) {
+        try {
+            OutputStream outputStream = httpExchange.getResponseBody();
+            httpExchange.getResponseHeaders().set("Content-Type", contentType);
+            httpExchange.sendResponseHeaders(responseCode, content.length());
+            outputStream.write(content.getBytes());
+            outputStream.flush();
+            outputStream.close();
+            return true;
+        } catch(java.io.IOException e) {
+            e.printStackTrace(System.out);
+            return false;
+        }
+    }
+
+    public static boolean writeHTTPResponse(HttpExchange httpExchange, int responseCode, String content) {
+        return writeHTTPResponse(httpExchange, responseCode, content, "text/plain; charset=us-ascii");
+    }
+}
+
+class BNFHandler implements HttpHandler {
+
+    Logger logger = Logger.getLogger(BNFHandler.class.getName());
+
+    public BNFHandler() {
+    }
```

Comment:
what's best practice here?  Is there a default constructor without this?

Reply:
not sure. but removing it.